src/containers/Settings/ducks.ts

```diff
@@ -0,0 +1,59 @@
+import update from 'immutability-helper'
```

Comment:
this file is deprecated and should not be added again. Use `useConfig` from `@apisuite/fe-base` to access settings.